packages/language/src/document.js

```diff
@@ -745,12 +964,19 @@ Document.formatSentiment_ = function(sentiment, verbose) {
  */
 Document.formatTokens_ = function(tokens, verbose) {
   if (!verbose) {
-    tokens = tokens.map(function(token) {
-      return {
-        text: token.text.content,
-        partOfSpeech: Document.PART_OF_SPEECH[token.partOfSpeech.tag],
-        partOfSpeechTag: token.partOfSpeech.tag
-      };
+    tokens = tokens.map(function(rawToken) {
+      var token = extend({}, rawToken.partOfSpeech, {
+        text: rawToken.text.content,
+        partOfSpeech: Document.PART_OF_SPEECH[rawToken.partOfSpeech.tag]
+      });
+
+      for (var part in token) {
+        if (token.hasOwnProperty(part) && /UNKNOWN/.test(token[part])) {
+          token[part] = false;
```

Comment:
I'm not sure if `false` or `undefined` is better here. `false` implies there was an answer, but `undefined` means "we don't know", which better matches `UNKNOWN`. What do you think?